test/e2e/l7networkpolicy_test.go

```diff
@@ -289,6 +293,71 @@ func testL7NetworkPolicyHTTP(t *testing.T, data *TestData) {
 	})
 }
 
+func testL7NetworkPolicyHTTPLargeResponse(t *testing.T, data *TestData) {
+	skipIfNumNodesLessThan(t, 2)
+	clientPodName := "test-l7-http-large-resp-client-selected"
+	clientPodLabels := map[string]string{"test-l7-http-large-resp-e2e": "client"}
+
+	// Create a client Pod, with the Pod being selected by the test L7 NetworkPolicy as target.
+	require.NoError(t, NewPodBuilder(clientPodName, data.testNamespace, agnhostImage).OnNode(nodeName(0)).WithLabels(clientPodLabels).Create(data))
+	_, err := data.podWaitForIPs(defaultTimeout, clientPodName, data.testNamespace)
+	require.NoError(t, err, "Expected IP for Pod '%s'", clientPodName)
+
+	// Create a hostNetwork server Pod as the destination selected by the test L7 NetworkPolicy, ensuring test traffic
+	// traverses antrea-gw0.
+	serverPodName := "test-l7-http-large-resp-server"
+	cmd := []string{"/agnhost", "netexec", "--http-port=8081"}
+	require.NoError(t, NewPodBuilder(serverPodName, data.testNamespace, agnhostImage).
+		WithHostNetwork(true).
+		OnNode(nodeName(1)).
+		WithCommand(cmd).
+		Create(data))
+	podIPs, err := data.podWaitForIPs(defaultTimeout, serverPodName, data.testNamespace)
+	require.NoError(t, err, "Expected IP for Pod '%s'", serverPodName)
+	serverIPs := podIPs.AsSlice()
+
+	l7ProtocolAllowsPathShell := []crdv1beta1.L7Protocol{
+		{
+			HTTP: &crdv1beta1.HTTPProtocol{
+				Path: "/shell*",
+			},
+		},
+	}
+	// Create a test egress L7NetworkPolicy allowing HTTP path "shell*".
+	policyAllowPathShellName := "test-l7-http-allow-path-shell"
+	createL7NetworkPolicy(t, data, false, policyAllowPathShellName, 1, nil, clientPodLabels, ProtocolTCP, p8081, l7ProtocolAllowsPathShell)
```

Comment:
use `"8081"` instead of `p8081`
I think using a constant provides no value here, especially when you consider the fact that you "hardcoded" `8081` at line 309

BTW, is there any reason to use port 8081 here while `testL7NetworkPolicyHTTP` uses port 8080?

Reply:
No special reason. Just because that the Pod is an hostNetwork one, and the port 8080 is potentially used by other services in the Node where it resides.